specification/schemas/return-reports/ReturnReportFilters.json

```diff
@@ -0,0 +1,194 @@
+      "description": "Array of organization IDs to filter included returns by.",
+      "items": {
+        "$ref": "#/components/schemas/Uuid"
+      },
+      "example": [
+        "35eddf50-1d84-47a3-8479-6bfda729cd99",
+        "a294ee55-bc94-4890-b734-afd56c158f95"
+      ]
+    },
+    "brokers": {
+      "type": "array",
+      "description": "Array of broker IDs to filter included returns by.",
+      "items": {
+        "$ref": "#/components/schemas/Uuid"
+      },
+      "example": [
+        "35eddf50-1d84-47a3-8479-6bfda729cd99",
+        "a294ee55-bc94-4890-b734-afd56c158f95"
+      ]
+    },
+    "outbound_shipment_identifier": {
+      "type": "string",
+      "description": "Filter entries by return order's outbound shipment tracking code.",
+      "example": "#123456"
+    },
+    "preferred_outcome": {
+      "type": "string",
+      "enum": [
+        "refund",
+        "exchange"
+      ],
+      "description": "Filter entries by return order item's preferred outcome.",
+      "example": "exchange"
+    },
+    "return_methods": {
+      "type": "array",
+      "description": "Array of return method IDs to filter included returns by.",
+      "items": {
+        "type": "string",
+        "format": "uuid",
+        "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$"
+      },
+      "example": [
+        "a294ee55-bc94-4890-b734-afd56c158f95",
+        "c8ac2fc8-2e75-4614-9e56-aa270d4338b0"
+      ]
+    },
+    "paid_at": {
+      "type": "object",
+      "description": "Either the `created_at`, `approved_at` filter or the `paid_at` filter is required for creating a report.",
+      "required": [
+        "date_from",
+        "date_to"
+      ],
+      "properties": {
+        "date_from": {
+          "type": [
+            "string",
+            "number"
+          ],
+          "description": "Date string in ISO 8601 format or unix timestamp. Only returns paid at >= this date will be in the report.",
+          "example": "2024-02-01"
+        },
+        "date_to": {
+          "type": [
+            "string",
+            "number"
+          ],
+          "description": "Date string in ISO 8601 format or unix timestamp. Only returns paid at <= this date will be in the response.",
+          "example": "2024-03-01"
+        }
+      }
+    },
+    "return_reasons": {
+      "type": "array",
+      "description": "Comma separated string of return reason codes to filter by. Only entries that correspond with one of the return reason codes will be in the report.",
+      "items": {
+        "type": "string",
+        "example": "product-damaged,product-is-defect"
+      }
+    },
+    "return_statuses": {
+      "type": "array",
+      "description": "Comma separated string of return status codes to filter by. Only entries that correspond with one of the return status codes will be in the report.",
+      "items": {
+        "type": "string",
+        "example": "return-pending,return-approved"
+      }
+    },
```

Comment:
I suspect something is going wrong here (with both `return_reasons` and `return_statuses`). It makes sense to have a comma separated string of codes/statuses, _or_ have an array of strings. Since this is a POST body and we don't need to put everything in the URL, I'd opt for the array of strings. So, instead of
```
"items": {
  "example": "return-pending,return-approved" <<<---- this is an example of a single array element
}
```
we get
```
"items": {
  "type": "string"
},
"example": [
  "return-pending",
  "return-approved"
]
```
you should also update the description to reflect these changes